### PR TITLE
Cli: update 'keysequence' to make keyswithguardtime configurable

### PR DESCRIPTION
**Pull request by librasungirl**
The motivation is for Test 6.6.2 (ED/SED) in which Harness only waits limited time for keyswitch's taking effect.

@jwhui please have a review. Thanks.

### CODE REVIEW

File: include/openthread.h [hunk 1]
```diff
@@ -1271,6 +1271,28 @@ uint32_t otGetKeySequenceCounter(otInstance *aInstance);
...
+ *
+ * @sa otSetKeySwitchGuardTime
+ */
+uint32_t otGetKeySwitchGuardTime(otInstance *aInstance);
+
+/**
+ * Set the thrKeySequenceCounter.
+ *
+ * @param[in]  aInstance            A pointer to an OpenThread instance.
+ * @param[in]  aGuardTime           The thrKeySwitchGuardTime value.
```
Comment (nibanks) [inline include/openthread.h]: I think it is worth updating the comments to indicate the units here (same for above). Also the comment parameter name doesn't match the actual parameter.

Comment (librasungirl) [inline include/openthread.h]: Done. Thanks @nibanks

File: include/openthread.h [hunk 2]
```diff
@@ -1271,6 +1271,28 @@ uint32_t otGetKeySequenceCounter(otInstance *aInstance);
...
+ *
+ * @sa otSetKeySwitchGuardTime
+ */
+uint32_t otGetKeySwitchGuardTime(otInstance *aInstance);
+
+/**
+ * Set the thrKeySequenceCounter.
+ *
+ * @param[in]  aInstance            A pointer to an OpenThread instance.
+ * @param[in]  aKeySwitchGuardTime           The thrKeySwitchGuardTime value (in hours).
```
Comment (jwhui) [inline include/openthread.h]: style: fix alignment

**Comment by jwhui** [comment]
@librasungirl, you'll need to update the `thread-cert` scripts to get them to pass.

**Pull request merged**